### ISSUE TITLE
Explore "Upload tool"

### ISSUE THREAD

**Issue by hofmannc** [MPDL/dataverse#20]
Testserver: https://prod-edmond2.mpdl.mpg.de/
Browser: Chrome / Firefox

Goal of the test: Find limits of upload items and check if error messages are sensible

Actions:

1. Log in -> "Add Data" -> "new Dataset" -> fill out at least mandatory fields -> add files by "select foles to add" and "drag and drop files here" -> "save Data set"

2. Open created data set in result list -> "upload files"
3. later: publish data set
    

Task:
Upload "exotic" sample data -> vary format (audio/video/text)/size/name/amount of files (but please inform Markus, if you are planning to upload really huge files :-) )

-> check if

    upload is complete and correct
    user is well informed (correct error message) if data format is not suitable/too big etc.
    files are acessible after download
    files are published completely and correctly

Subquestions on the task:

    filter files by name/size/types -> correctly sorted?
    Change file names and check what happens in filter list and re-upload
    experiment with private and released files -> e.g. are they correctly deaccessioned/deleted?

Please report observations in a listing (e.g. in Word/Excel or in this ticket) if something unexpected happens. Please follow this form:

Browser: 
User: admin? regular user?

Actions:
Observation: 
Expected result:

**Comment by HoMPDL** [comment]
Ticket für EDMOND
1.
Browser: FF
User: regular
Actions: assign roles to Users/Groups for dataset
Observation: 
•	Hard to figure out names of groups/ which groups exist (Are they listed somewhere?)
•	Unclear, which name of single persons to fill in (e.g. Full name? @name? E-mail?)
•	Many User names already on the list without connection to the dataset (e.g. Mark Barber)
•	ID Name exceeds the box
•	after removing an user, the number of users with permission doesn’t immediately update, but after refreshing the page
Expected result: 
•	maybe remark that the name is enough
•	Link to existing groups?
•	Right numbers of users
2.
Browser: FF
User: regular
Actions: granting access to users/groups for restricted files
Observation:
•	Restricted files can’t be found  files can’t be assigned to specific users
•	Restricted files are displayed after ticking ‘Include Deleted Files’
Expected Result:
•	Restricted files are displayed 
3.
Browser: FF
User: regular
Actions: removing roles from certain users for dataset restriction
Observation: 
•	It is possible to remove the curator, is it intended?
4.
Browser: FF
User: regular
Actions: Deaccession dataset
Observation: 
•	After deaccession dataset you cannot restore the files directly
•	However, it is possible to add new files to the dataset in order to access the files again. Using this simple technique, it is possible to restore the whole set again. 
•	After publishing the new files, the deaccession button isn’t available. Only after refreshing the page.
Expected result: 
•	Either the dataset is not accessible at all or adding a undone button, which enables public access again
5. 
Browser: FF
User: regular
Action: downloading a public file from a dataset, where I deleted myself as the curator
Observation: 
•	The Author (who obviously has no longer access to the files) was contacted, even though the file is public 
Expected result: 
•	Public file can be downloaded without authors request
6.
Browser: FF
User: regular
Actions: Loading a thumbnail image
Observation:
•	the loading icon doesn’t vanish after successfully uploading the image, but rather keeps indicating the loading process is still going on  user may wait a long time before realising the upload is already done
•	the icon vanishes after saving Changes 
•	thumbnail image was deleted after weekend
7.
Browser: FF
User: regular
Actions: Giving permissions to users
Observation: 
•	ManageFilePermission is still a tag for admin and curator, even though this option isn’t available anymore
8.
Browser: FF
User: regular
Actions: Giving permissions to users
Observations:
•	Other unaccompanied users were removed but added users (in this example Yves) still exceed the box in their ID column
